### ISSUE TITLE
Is this repository being maintained?

### ISSUE THREAD

**Issue by pelizza** [microsoft/azure-devops-extension-api#53]
Hi there,

I have been using the Azure DevOps API and SDK for a while now, as they are the latest tools for implementing Azure DevOps extensions.

It lacks a lot of documentation; sometimes I am able to figure out what I need directly from the code, but other times developers just need some documentation or usage examples to be able to use it. In these days that we talk a lot about Developer Experience, I can say it hasn't been a very good experience so far.

Additionally, there are quite a few issues/questions open - I have opened a couple of them myself - but they just don't get an answer.

Can we have a transparent update on how this repository is maintained, and what is the roadmap in terms of documentation and support?

Thank you.

**Comment by FlexiGit** [comment]
The same question applies to the other extension related repos: azure-devops-extension-sdk, azure-devops-ui and azure-devops-extension-sample.

**Comment by aleqsss** [comment]
I agree, I am wondering the same thing. It's somewhat of a guessing game trying to use this. I don't know how many hours I've wasted on trivial things here.

More examples, and awareness of maintenance. I'm also curios of the roadmap here.

**Comment by jeffpriz** [comment]
Yeah, it is frustrating, I've built several Pipeline Task extensions.. and I'm currently trying to work through a UI Web Hub extension.. but there is NO example (no modern example at least) on how to build a UI extension that calls a REST api.. 

There are so many thins out there... this azure-devops-extension-api repo, then also the SDK repo
https://github.com/Microsoft/azure-devops-extension-sdk
The Node API Repo:
https://github.com/microsoft/azure-devops-node-api/
The extension sample repo:

**Comment by karelkral** [comment]
Agree with the previous notes. Developing for Azure DevOps is the most frustrating thing in my 25-year developer life.

**Comment by jan-machacek** [comment]
I am very disappointed with MS approach to these devops products :-( We want to use something from new api versions and it is make me crazy looking for solutions and again and again.... MS Docs is for old VSS, new one is without support, really perfect product :-/.
